demo/demo/tests/conftest.py

```diff
@@ -14,9 +14,3 @@ def user(db):
 def user_client(client, user) -> Client:
     client.force_login(user)
     return client
-
-
-@pytest.fixture(autouse=True)
-def clear_cache():
```

Comment:
idk if we need this here, but just wanted to mention it:
https://github.com/pytest-dev/pytest-django/issues/527#issuecomment-1115887487

Reply:
I believe we can simply clear caches in our case for simplicity

Reply:
Django's caches value defaults to this:

```python
{'default': {'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'}}
```

Which should be sufficient for our use case.

Speaking of `xdist`, I don't think it would be a problem here, cause we don't need access to share cache across multiple test cases.